src/main/java/me/magnet/consultant/ServiceIdentifier.java

```diff
@@ -123,6 +138,7 @@ public String toString() {
 		getDatacenter().ifPresent(dc -> descriptors.add("dc=" + dc));
 		getHostName().ifPresent(host -> descriptors.add("host=" + host));
 		getInstance().ifPresent(instance -> descriptors.add("instance=" + instance));
+		getTags().stream().forEach((tag) -> descriptors.add("tag=" + tag));
```

Comment:
If a service has multiple tags, this becomes something like `oauth[dc=eu-central,host=docker0,instance=<uuid>,tag=192,tag=staging,tag=testing]` etc. Wouldn't it be far nicer (and easier to parse) something like: `oauth[dc=eu-central,host=docker0,instance=<uuid>,tags=192,staging,testing]`.